drivers/storage/portworx/component/securitycontextconstraints.go

```diff
@@ -99,12 +105,20 @@ func (s *scc) Reconcile(cluster *opcorev1.StorageCluster) error {
 			}
 		} else if err == nil {
 			resourceVersion := existingScc.ResourceVersion
-			pxutil.CleanupObject(&scc)
-			pxutil.CleanupObject(&existingScc)
-			logrus.Infof("Updating %s security context constraints", scc.Name)
-			if !reflect.DeepEqual(scc, existingScc) {
+			sccPriority := *scc.Priority
+			existingSccPriority := *existingScc.Priority
+			sccAllowPrivilegeEscalation := *scc.AllowPrivilegeEscalation
+			existingSccAllowPrivilegeEscalation := *existingScc.AllowPrivilegeEscalation
+			cleanupSccObject(&scc)
+			cleanupSccObject(&existingScc)
+			if sccPriority != existingSccPriority ||
+				sccAllowPrivilegeEscalation != existingSccAllowPrivilegeEscalation ||
+				!equality.Semantic.DeepDerivative(scc, existingScc) {
```

Comment:
could you change this to deepEqual. deepDerivative will not compare values that are nil in first parameter, it is not a good fit for this case as we care comparing the object entirely.

Reply:
Yes, I'll change that